packages/visual-dev/src/components/Switch/Switch.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import styled from 'styled-components'
+
+import * as Styles from './Styles'
+
+interface SwitchProps {
+	id: string
+	status?: "on" | "off"
+	color?: "success" | "critical"
+}
+
+export const Switch: React.FC<SwitchProps> = ({
+	id,
+	color = "success",
+	status = "off"
+}) => {
+	return (
+		<div>
+			<SwitchDiv>
+				<SwitchBox color={color} id={id} type="checkbox" defaultChecked={status=="on" ? true : false}/>
+				<SwitchLabel htmlFor={id}/>
+			</SwitchDiv>
+		</div>
+	)
+}
+
```

Comment:
Nothing technically wrong here, however, as a general convention we usually put styled-component declarations before their usage in a view.